### PR TITLE
[Doc Improvement][Updated bot id description][3102800]

### PR DESCRIPTION
**Pull request by v-rmvv**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f8de6639104775315e70b7712e7c9f5339d1daaf/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 93 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ac7527ea-6c3a-4972-b741-112fa41322c3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0044f0e56365e15a2c8fd0c68d1bf7b545d12754/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 93 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de3f6e48-33da-4103-aa23-d9d8fada8450) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5ca5cd5166b3b150b414af096559f74a392e7da9/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 93 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c07cbac3-ad5e-47d7-9c69-bad718ed8a26) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Meghana-MSFT — APPROVED** [review]
ok

File: msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ You need:
     > [!IMPORTANT]
     >
     > - **Standalone bot**: If you're building a standalone bot, enter the application ID URI as api://botid-{YourBotId}. Here, {YourBotId} is your Azure AD application ID.
-    > - **Application ID URI for app with multiple capabilities**: If you're building an app with a bot, a messaging extension, and a tab, enter the application ID URI as `api://fully-qualified-domain-name.com/BotId-{YourClientId}`, where the BotId is your bot app ID.
+    > - **Application ID URI for app with multiple capabilities**: If you're building an app with a bot, a messaging extension, and a tab, enter the application ID URI as `api://fully-qualified-domain-name.com/botId-{YourClientId}`, where `{YourClientId}` is your bot app ID.
```
Comment (yuezk) [inline msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md:160]: After reading so many docs, I believe `botId` should be `botid`, though I don't know whether `botId` could work as well.

Comment (v-ypalikila) [reply msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md]: Hi @yuezk ,

We've updated `botId` to `botid`.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6ac63aa974ad02c643f87afd78317e2a1bed7964/msteams-platform%2Fbots%2Fhow-to%2Fauthentication%2Fbot-sso-register-aad.md) | 93 | 32 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f6009458-c28c-4773-92e0-a2f2c057e627) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md [hunk 2]
```diff
@@ -518,7 +518,7 @@ To configure scope and authorize trusted client applications, you'll need:
     > [!IMPORTANT]
     >
     > - **Standalone bot**: If you're building a standalone bot, enter the application ID URI as api://botid-{YourBotId}. Here, {YourBotId} is your Azure AD application ID.
-    > - **Application ID URI for app with multiple capabilities**: If you're building an app with a bot, a messaging extension, and a tab, enter the application ID URI as `api://fully-qualified-domain-name.com/BotId-{YourClientId}`, where the BotID is your bot app ID.
+    > - **Application ID URI for app with multiple capabilities**: If you're building an app with a bot, a messaging extension, and a tab, enter the application ID URI as `api://fully-qualified-domain-name.com/botId-{YourClientId}`, where `{YourClientId}` is your bot app ID.
```
Comment (yuezk) [inline msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md:521]: What about this `botId`?

Comment (v-ypalikila) [reply msteams-platform/bots/how-to/authentication/bot-sso-register-aad.md]: @yuezk , Updated both the instances.

**Review by yuezk — APPROVED** [review]
looks good

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Pull request merged**